'cachePackages.js' is missing

I found the following error message in my log:

`{"message":"Plugin failed to load","stack":["Error: Cannot find module '/Users/taowang/.zazu/plugins/tinytacoteam/zazu-package-manager/cachePackages.js'","    at Module._resolveFilename (module.js:455:15)","    at Function.Module._resolveFilename (/Applications/Zazu.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)","    at Function.Module._load (module.js:403:25)","    at Module.require (module.js:483:17)","    at require (internal/module.js:20:19)","    at module.exports (/Applications/Zazu.app/Contents/Resources/app.asar/lib/freshRequire.js:3:10)","    at new ServiceScript (/Applications/Zazu.app/Contents/Resources/app.asar/blocks/external/serviceScript.js:18:22)","    at /Applications/Zazu.app/Contents/Resources/app.asar/packages/plugin.js:82:28","    at Array.forEach (native)","    at /Applications/Zazu.app/Contents/Resources/app.asar/packages/plugin.js:78:30"],"plugin":"tinytacoteam/zazu-package-manager","block":"Cache Packages","level":"error","timestamp":"2017-02-09T13:31:30.414Z"}`

It said cannot find module `tinytacoteam/zazu-package-manager/cachePackages.js`, which defined in the `zazu.json`:

```json
      {
        "id": "Cache Packages",
        "type": "ServiceScript",
        "script": "cachePackages.js",
        "interval": 1800000,
        "interval.comment": "30_minutes"
      }
```

And I found it's missing from the repo.

--- a/zazu.json
+++ b/zazu.json
@@ -5,7 +5,7 @@
       {
         "id": "Cache Packages",
         "type": "ServiceScript",
-        "script": "cachePackages.js",
+        "script": "src/refresh.js",
         "interval": 1800000,
         "interval.comment": "30_minutes"
       }

